docker-compose.yml

```diff
@@ -12,6 +12,8 @@ services:
       - 3306:3306
     networks:
       - backend
+    volumes:
+      - /opt/xskillz/data:/var/lib/mysql  
```

Comment:
You could change path before ':' to your host path directory where you would like to persist your data.